src/Language/Spectacle/Syntax/Modal/Rewrite.hs

```diff
@@ -0,0 +1,337 @@
+{-# LANGUAGE TypeFamilies #-}
+
+module Language.Spectacle.Syntax.Modal.Rewrite
```

Comment:
This module and the rewrite rules being expressed in it also need documentation to help readers unfamiliar with these rewrite rules understand how normalization will behave for their specifications.